src/server/game/Entities/Player/Player.h

```diff
@@ -1,4 +1,4 @@
-﻿/*
+/*
```

Comment:
?? bom character?

Reply:
I remember chinese contributors had this issue with invisible "BOM" character added by default by MVS

Reply:
I couldn't solve this.

Reply:
I had a solution but it's a bit tedious, requires to run a command to convert the file before sending the PR

Reply:
Quote from the buddy @Rochet2 "its removing bom it seems, old name is byte order mark (BOM) and new name is ZERO WIDTH NO-BREAK SPACE"